clusters/app.ci/prow/03_deployment/adapter_imagestreams.yaml

```diff
@@ -53,6 +53,19 @@ items:
       from:
```

Comment:
Can we add them to just the arm cluster? They just occupy space on all the other clusters

Reply:
good point. will update.